Sources/Sentry/NSData+Sentry.m

```diff
@@ -0,0 +1,15 @@
+#import "NSData+Sentry.h"
+
+@implementation NSData (Sentry)
+
+- (NSData *)nullTerminated
+{
+    if (self == nil) {
+        return nil;
+    }
+    NSMutableData *mutable = [NSMutableData dataWithData:self];
+    [mutable appendBytes:"\0" length:1];
+    return mutable;
+}
+
+@end
```

Comment:
why do we need that?

Reply:
It was like this in SentryCrash. I'm not sure if we need it. I will figure it out and add a comment if we have to keep it.

Reply:
I added a comment.

Reply:
comment is resolved, but theres no comment in the code, so why do we need it?

Reply:
nvm its in the .h clas